### PR TITLE
[Method References 2] LPS-164454 Convert method reference to ServiceTrackerMap in module dynamic-data-mapping-api

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I have inlined private methods.

Detial: https://github.com/liferay-core-infra/liferay-portal/pull/1926

### CODE REVIEW

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by dantewang** [comment]
@W2jx Conflicts, please update.

**Comment by W2jx** [comment]
Force-Push is just to resolve the conflict about semantic version.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistry.java [hunk 1]
```diff
@@ -34,45 +34,34 @@
 public class DDMDisplayRegistry {
 
 	public DDMDisplay getDDMDisplay(String portletId) {
-		return _getDDMDisplay(portletId);
+		return _serviceTrackerMap.getService(portletId);
 	}
 
 	public List<DDMDisplay> getDDMDisplays() {
-		return _getDDMDisplays();
+		return ListUtil.fromCollection(_serviceTrackerMap.values());
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistry.java:41]: @W2jx The `getDDMDisplays` method is not used, please remove it.

**Comment by dantewang** [comment]
Closing, please resend.

**Pull request closed without merging**